x-pack/plugins/ingest_pipelines/public/application/components/pipeline_processors_editor/components/shared/map_processor_type_to_form.tsx

```diff
@@ -121,69 +124,123 @@ export const mapProcessorTypeToDescriptor: MapProcessorTypeToDescriptor = {
     label: i18n.translate('xpack.ingestPipelines.processors.label.enrich', {
       defaultMessage: 'Enrich',
     }),
+    description: () => (
+      <FormattedMessage
+        id="xpack.ingestPipelines.processors.description.enrich"
+        defaultMessage="Adds enrich data to incoming documents based on an {enrichPolicyLink}."
+        values={{
+          enrichPolicyLink: (
+            <EuiLink external target="_blank" href={esDocUrl + '/ingest-enriching-data.html'}>
+              {'enrich policy'}
+            </EuiLink>
+          ),
+        }}
+      />
+    ),
```

Comment:
```suggestion
   description: function Description() {
      const {
        services: { documentation },
      } = useKibana();
      const esDocUrl = documentation.getEsDocsBasePath();
      return (
        <FormattedMessage
          id="xpack.ingestPipelines.processors.description.enrich"
          defaultMessage="Adds enrich data to incoming documents based on an {enrichPolicyLink}."
          values={{
            enrichPolicyLink: (
              <EuiLink external target="_blank" href={esDocUrl + '/ingest-enriching-data.html'}>
                {'enrich policy'}
              </EuiLink>
            ),
          }}
        />
      );
    }
```

You can convert this into a named function component and `useKibana` to get the esDocUrl. You will need to add `import { useKibana } from '../../../../../shared_imports';` directly after your EUI import at the top of the file

Reply:
Makes sense. Thanks for your help! 🙇